database/database.go

```diff
@@ -457,3 +460,29 @@ func (vdb *VspDatabase) CheckIntegrity(ctx context.Context, params *chaincfg.Par
 
 	return nil
 }
+
+// UpdateAdminPass stores admin password hash in Database.
+func (vdb *VspDatabase) UpdateAdminPass(adminPassHash string) error {
+return vdb.db.Update(func(tx *bolt.Tx) error {
+		vspBkt := tx.Bucket(vspBktK)
+
+		// Add admin password hash to database.
+return vspBkt.Put(adminPassHashK, []byte(adminPassHash))
+	})
+}
+
+// GetAdminHash returns admin password hash from Database.
+func (vdb *VspDatabase) GetAdminHash() ([]byte, error) {
+	var adminPassHash []byte
+	err := vdb.db.View(func(tx *bolt.Tx) error {
+		vspBkt := tx.Bucket(vspBktK)
+
+		adminPassHash = vspBkt.Get(adminPassHashK)
```

Comment:
No need for the space between.
```suggestion
		vspBkt := tx.Bucket(vspBktK)
		adminPassHash = vspBkt.Get(adminPassHashK)
```